nucliadb/nucliadb/writer/api/v1/export_import.py

```diff
@@ -138,10 +137,13 @@ async def start_export_task(context: ApplicationContext, kbid: str, export_id: s
         raise
 
 
-async def start_import_task(context: ApplicationContext, kbid: str, import_id: str):
+async def start_import_task(
+    context: ApplicationContext, kbid: str, import_id: str, import_size: int
```

Comment:
Should be this `max_import_size`?

```suggestion
    context: ApplicationContext, kbid: str, import_id: str, max_import_size: int
```

Reply:
no, it´s the actual size of the import blob